packages/wrangler/templates/d1-beta-facade.js

```diff
@@ -0,0 +1,174 @@
+// src/shim.ts
+import worker from "__ENTRY_POINT__";
+export * from "__ENTRY_POINT__";
+
+// src/index.ts
+var D1Database = class {
```

Comment:
`var`?

Why not just

```
class D1Database
```

Reply:
this is the built output of an internal project, so I don't have too much control over stylistic things like that.